portfolio/src/main/webapp/script.js

```diff
@@ -45,12 +45,38 @@ content.appendChild(
     link: "https://vedantroy.github.io/",
   })
 );
-fetch("/data")
-  .then((resp) => resp.json())
-  .then((commentsArray) => {
-    for (const comment of commentsArray) {
-      const p = document.createElement("p");
-      p.textContent = comment;
-      document.body.append(p);
+
+let maxComments = 3;
```

Comment:
I'm not as familiar with JavaScript best practices and constraints, but should maxComments be an argument to getComments() instead of a global?